src/vega.schema.ts

```diff
@@ -1183,3 +1184,22 @@ export interface VgTitleConfig {
    */
   orient?: TitleOrient;
 }
+
+export type VgComparatorOrder = 'ascending' | 'descending';
+
+export interface VgComparator {
+  field?: string | string[];
+  order?: VgComparatorOrder;
```

Comment:
shouldn't this allow array too?

Reply:
Assuming this is referring to the field parameter, it is resolved.